pkg/splunk/enterprise/standalone.go

```diff
@@ -229,3 +239,20 @@ func validateStandaloneSpec(cr *enterpriseApi.Standalone) error {
 
 	return validateCommonSplunkSpec(&cr.Spec.CommonSplunkSpec)
 }
+
+// helper function to get the list of Standalone types in the current namespace
+func getStandaloneList(c splcommon.ControllerClient, cr splcommon.MetaObject, listOpts []client.ListOption) (int, error) {
```

Comment:
As we are not using in the code, may be move it to test code?

Reply:
Ditto, it was left intentionally here